<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>E-Commerce Marketplace</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <!-- Login Page -->
      <widget class="QWidget" name="loginPage">
       <layout class="QVBoxLayout" name="loginLayout">
        <item>
         <widget class="QLabel" name="loginLabel">
          <property name="text">
           <string>Login Page</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="loginEmail">
          <property name="placeholderText">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="loginPassword">
          <property name="placeholderText">
           <string>Password</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showRegisterButton">
          <property name="text">
           <string>Create Account</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>

      <!-- Register Page -->
      <widget class="QWidget" name="registerPage">
       <layout class="QVBoxLayout" name="registerLayout">
        <item>
         <widget class="QLabel" name="registerLabel">
          <property name="text">
           <string>Register Page</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="registerEmail">
          <property name="placeholderText">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="registerPassword">
          <property name="placeholderText">
           <string>Password</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="registerButton">
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showLoginButton">
          <property name="text">
           <string>Back to Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>

      <!-- Main Page -->
      <widget class="QWidget" name="mainPage">
       <layout class="QVBoxLayout" name="mainLayout">
        <item>
         <widget class="QLabel" name="mainLabel">
          <property name="text">
           <string>Welcome to the Marketplace!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Browse Products</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cartButton">
          <property name="text">
           <string>View Cart</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="logoutButton">
          <property name="text">
           <string>Logout</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
